src/diff/props.js

```diff
@@ -122,5 +122,6 @@ function setProperty(dom, name, value, oldValue, isSvg) {
  * @private
  */
 function eventProxy(e) {
-	return this._listeners[e.type](options.event ? options.event(e) : e);
+	this._listeners[e.type](options.event ? options.event(e) : e);
```

Comment:
Do we need to still capture the return value from the handler and return it?

Reply:
I don't think we do. We attach `eventProxy` like a standard event handler and those don't return anything:

```
dom.addEventListener("input"; e => {
  // No need to return anything here
});
```